add templates to the ci signal handbook folder

leonardpahlke opened the pull request:
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

In the past documents have been past from release to release without documenting the format. This PR adds three templates to the ci signal handbook folder. 

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

cc  @JamesLaverack @jrsapi

leonardpahlke commented:
maybe instead of uploading the excel sheet as template give an example of the stucture of the excel sheet in markdown file? (opinions?)

something like this:
In the previous release cycled we used a excel sheet to keep track of status reporting in release team meetings and ... 

The first sheet is used to  ... and looked like this:

| Release  | Planned Date | Branch Manager | Ci Signal Member | Testgrid boards to observe |
| ---   | ---   | ---    | ---       |  ---     |
| 1.24.0-alpha.2   | ...   | ...    | ...       | ...      |

...

leonardpahlke commented:
One of the template which I add with this PR defines the weekly ci signal report which gets posted to slack & k/dev.
I'm thinking of generating the entire weekly CI signal report (see [project board] (https://github.com/orgs/kubernetes/projects/65#card-76512722)) - this would automatically "fill in" the template I've added here.

leonardpahlke commented:
/hold

there is currently a discussion ongoing about templating (see [slack discussion](https://kubernetes.slack.com/archives/CN0K3TE2C/p1648500977041789))

mkorbi commented on release-team/role-handbooks/ci-signal/template-ci-signal-shadow-onboarding-agenda.md:
```diff
@@ -0,0 +1,87 @@
+Job comes down to:
+
+
+1. Monitor testgrid: Particularly interested the [testgrid dashboards](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#blocking-vs-informing-dashboards): [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) & [master-informing](https://testgrid.k8s.io/sig-release-master-informing).
```
this part is redundant to another PR https://github.com/kubernetes/sig-release/pull/1869
as well as the [overview](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/ci-signal#overview-of-ci-signal-responsibilities) section of the handbook

leonardpahlke replied:
yes! I think we can reference it as a place holder so we have a single source of truth!

leonardpahlke replied:
yes! I think we can reference it as a place holder so we have a single source of truth!

mkorbi reviewed (commented):
+1 for templates, I think it would be good to have them as gdocs. Not sure how they are handled, doe K8s/K8s-Sig-Release have a folder that could be used for it? Maybe also for other teams relevant?

@saschagrunert @JamesLaverack

saschagrunert commented:
> +1 for templates, I think it would be good to have them as gdocs. Not sure how they are handled, doe K8s/K8s-Sig-Release have a folder that could be used for it? Maybe also for other teams relevant?

In Google Docs? No, we usually track all data like this in GitHub which is what I would prefer as well. What we could do is collecting Google Docs in a Markdown document referenced from this repository.

leonardpahlke commented:
PR, updated :) 
I have removed some of the templates that I originally added as they are no longer needed. 
1. the Excel file is no longer needed as a template because we use the [CI Signal GitHub project board](https://github.com/orgs/kubernetes/projects/68/views/11) for scheduling reports.
2. the onboarding template is probably redundant if we still add an "onpager" (see [PR](https://github.com/kubernetes/sig-release/pull/1869)) 

/unhold

leonardpahlke commented:
@mkorbi I updated the PR to add the template for the weekly ci signal report, WDYT?

mkorbi reviewed (approved):
lgtm
